Mage.Sets/src/mage/cards/g/GenesisOfTheDaleks.java

```diff
@@ -0,0 +1,45 @@
+package mage.cards.g;
+
+import java.util.UUID;
+
+import mage.abilities.common.SagaAbility;
+import mage.abilities.dynamicvalue.common.CountersSourceCount;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.constants.SagaChapter;
+import mage.constants.SubType;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.counters.CounterType;
+import mage.game.permanent.token.DalekToken;
+
+/**
+ *
+ * @author anonymous
+ */
+public final class GenesisOfTheDaleks extends CardImpl {
+
+    public GenesisOfTheDaleks(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{4}{B}{B}");
+        
+        this.subtype.add(SubType.SAGA);
+
+        // (As this Saga enters and after your draw step, add a lore counter. Sacrifice after IV.)
+        SagaAbility sagaAbility = new SagaAbility(this, SagaChapter.CHAPTER_III);

```

Comment:
Must be IV here (Sacrifice after IV):
`new SagaAbility(this, SagaChapter.CHAPTER_IV)`

Reply:
Yes, I will be implementing chapter IV once I-III work correctly.  For the time being, though, the reason why I created this draft is that the token creation on chapter I is not working and I was asked by @xenohedron to create it so they could check out the code.  It's not working even though ```CountersSourceCount(CounterType.LORE)``` works perfectly with another card (although for a different effect).